google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -27,7 +27,6 @@
 import com.google.api.gax.grpc.ChannelProvider;
 import com.google.api.gax.grpc.ExecutorProvider;
 import com.google.api.gax.grpc.InstantiatingExecutorProvider;
-import com.google.auth.Credentials;
```

Comment:
Probably the documentation about credentials on the line 75 about credentials should also be removed/modified.
(Writing here, since I could not find a way to add a comment on a line which was not modified/added/removed)